### PR TITLE
Added tests for current version and fixed some bug

### PR DESCRIPTION
**Pull request by lichunqiang**
[Build Status](https://travis-ci.org/lichunqiang/GB2260.php)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+phpunit.xml
+vendor
+build
+composer.lock
```
Comment (lepture) [inline .gitignore]: new line at end of file

Comment (lichunqiang) [inline .gitignore]: @lepture  I used editorcofig and added it to project

```
# editorconfig.org
root = true

# Unix-style newlines with a newline ending every file
[*]
end_of_line = lf
insert_final_newline = true
charset = utf-8

[*.php]
indent_style = space
indent_size = 4
trim_trailing_whitespace = true

[*.{js,twig,css,scss,html}]
indent_style = space
indent_size = 2

[{Gruntfile.js,bower.json,composer.json,package.json,*.yml}]
indent_style = space
indent_size = 2

[*.md]
trim_trailing_whitespace = false

```

But online is not display...

So we do not care about this issue :smile:

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # GB/T 2260
 
+[![Build Status](https://travis-ci.org/lichunqiang/GB2260.php.svg)](https://travis-ci.org/lichunqiang/GB2260.php)
```
Comment (gaomd) [inline README.md]: The URL should be `...cn/GB2260.php...`

@lepture @shen2 Please enable `GB2260.php` project at https://travis-ci.org/profile/cn, I don't have permission to do that as a Collaborator.

File: tests/GBTest.php [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace cn\tests;
+
+use cn\GB2260;
+
+class GBTest extends \PHPUnit_Framework_TestCase
```
Comment (gaomd) [inline tests/GBTest.php]: The class name should be `GB2260Test`, which is in connection with `GB2260.php`.

Comment (lichunqiang) [inline tests/GBTest.php]: @gaomd Ok, got it.

File: src/GB2260.php [hunk 4]
```diff
@@ -15,9 +15,11 @@ public static function getData()
 
     public static function parse($code)
     {
+        //@codeCoverageIgnoreStart
```
Comment (gaomd) [inline src/GB2260.php]: What's the reason to skip code coverage?

This code block was duplicated in `getData()` method, too.

Comment (lichunqiang) [inline src/GB2260.php]: Because I have no idea to write a case to cover this.
So i ignored.

File: README.md (hunk 2 above)
Comment (lepture) [inline README.md]: @gaomd done

Comment (lepture) [inline README.md]: @gaomd you are owner now.

File: src/GB2260.php (hunk 4 above)
Comment (gaomd) [inline src/GB2260.php]: By using code coverage we increase code quality, not pursuit 100%, please remove code coverage ignore annotations, we'll figure out a way to cover that later :)

And please [squash](https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit) commits in this pull request, then I'll merge it in a couple of days.

**Comment by lichunqiang** [comment]
@gaomd  done.

**Pull request merged**